src/cascadia/TerminalApp/userDefaults.json

```diff
@@ -6,6 +6,10 @@
 
     "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
 
+    // If enabled, selections are automatically copied to your clipboard
+    // To learn more about global settings, visit https://aka.ms/terminal-documentation
+    "copyOnSelect": false,
+
     "profiles":

```

Comment:
I think we should link to the documentation above every major section, so profiles, color schemes, and key bindings.